src/sentry/snuba/discover.py

```diff
@@ -961,7 +961,7 @@ def get_pagination_ids(event, query, params, organization, reference_event=None,
     # to use the new function values
     query = transform_deprecated_functions_in_query(query)
 
-    snuba_filter = get_filter(query, params)
+    snuba_filter = get_filter(query, params, adjust_interval=1)
```

Comment:
Not quite sure what the behavior here should be so I haven't written the test for it yet.

**Also: It doesn't look like this discover function is used anywhere outside of tests.**

Reply:
@wmak Didn't you remove this in an earlier PR?

Reply:
Ah yeah I think this was being used for event details pagination. Is safe to remove :+1: